tests/managed/subscripts.cs

```diff
@@ -0,0 +1,578 @@
+﻿﻿using System.Collections.Generic;
```

Comment:
Let's remove this file, it should be generated as part of msbuild with `Mono.TextTransform`.